web/src/enterprise/campaigns/detail/changesets/ExternalChangesetInfoCell.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import {
+    ExternalChangesetFields,
+    ChangesetExternalState,
+    ChangesetPublicationState,
+} from '../../../../graphql-operations'
+import { Observer } from 'rxjs'
+import { LinkOrSpan } from '../../../../../../shared/src/components/LinkOrSpan'
+import ExternalLinkIcon from 'mdi-react/ExternalLinkIcon'
+import { ChangesetLabel } from './ChangesetLabel'
+import { Link } from '../../../../../../shared/src/components/Link'
+import { ChangesetLastSynced } from './ChangesetLastSynced'
+
+export interface ExternalChangesetInfoCellProps {
+    node: ExternalChangesetFields
+    viewerCanAdminister: boolean
+    campaignUpdates?: Pick<Observer<void>, 'next'>
+}
+
+export const ExternalChangesetInfoCell: React.FunctionComponent<ExternalChangesetInfoCellProps> = ({
+    node,
+    viewerCanAdminister,
+    campaignUpdates,
+}) => (
+    <div className="d-flex flex-column">
+        <div className="m-0 mb-2">
+            <h3 className="m-0 d-inline">
+                <LinkOrSpan
+                    /* Deleted changesets most likely don't exist on the codehost anymore and would return 404 pages */
+                    to={
+                        node.externalURL && node.externalState !== ChangesetExternalState.DELETED
+                            ? node.externalURL.url
+                            : undefined
+                    }
+                    target="_blank"
+                    rel="noopener noreferrer"
+                >
+                    {node.title}
+                    {node.externalID && <>(#{node.externalID}) </>}
```

Comment:
Noticed this while trying it out locally:

```suggestion
                    {node.externalID && <> (#{node.externalID}) </>}
```